lms/djangoapps/courseware/views/index.py

```diff
@@ -182,6 +186,22 @@ def _verify_position(self):
             except ValueError:
                 raise Http404(u"Position {} is not an integer!".format(self.position))
 
+    def _create_next_param(self):
+        """
+        create the _next parameter for the staff user
+        """
+        if self.chapter_url_name and self.section_url_name:
+            _next = reverse(
+                'courseware_section',
+                kwargs={
+                    'course_id': unicode(self.course_key),
+                    'chapter': self.chapter_url_name,
+                    'section': self.section_url_name,
```

Comment:
You should really use [`courseware_position`](https://github.com/edx/edx-platform/blob/master/lms/urls.py#L482-L489)